# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1481895405 18000
#      Fri Dec 16 08:36:45 2016 -0500
# Branch stable
# Node ID 13881569f613cce6a15947ae42e3bce203c9245f
# Parent  544dac72d414f565ea863b14961e1fc7380e31cc
fix typo in liboctave version info (bug #49860)

* liboctave/module.mk: Use $(liboctave_liboctave_age), not
$(liboctave_age) to define liboctave_liboctave_version_info.
From Paul Irofti.

diff --git a/liboctave/module.mk b/liboctave/module.mk
--- a/liboctave/module.mk
+++ b/liboctave/module.mk
@@ -78,17 +78,17 @@ liboctave_liboctave_la_LIBADD += \
   libgnu/libgnu.la \
   $(LIBOCTAVE_LINK_DEPS)
 
 # Increment these as needed and according to the rules in the libtool manual:
 liboctave_liboctave_current = 4
 liboctave_liboctave_revision = 0
 liboctave_liboctave_age = 0
 
-liboctave_liboctave_version_info = $(liboctave_liboctave_current):$(liboctave_liboctave_revision):$(liboctave_age)
+liboctave_liboctave_version_info = $(liboctave_liboctave_current):$(liboctave_liboctave_revision):$(liboctave_liboctave_age)
 
 liboctave_liboctave_la_LDFLAGS = \
   -version-info $(liboctave_liboctave_version_info) \
   $(NO_UNDEFINED_LDFLAG) \
   @XTRA_CRUFT_SH_LDFLAGS@ \
   -bindir $(bindir) \
   $(LIBOCTAVE_LINK_OPTS) \
   $(WARN_LDFLAGS)
